package.json

```diff
@@ -3,12 +3,12 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build": "webpack --config ./webpack.config.prod.js --progress --colors",
-    "start": "npm-run-all --parallel startServer startJson",
-    "startServer": "webpack-dev-server --config ./webpack.config.dev.js",
-    "startJson": "json-server --watch ./src/db.json --port 3006",
-    "startTemplate": "VERSION=Template node ./setup.js",
-    "startProd": "VERSION=Prod node ./setup.js",
+    "start": "VERSION=Dev npm-run-all --parallel startBackendDev buildFrontend",
+    "buildFrontend": "webpack",
+    "startTemplate": "VERSION=Template npm-run-all --parallel buildFrontend startBackend",
+    "startProd": "VERSION=Prod npm-run-all -s buildFrontend startBackend",
+    "startBackendDev": "nodemon --watch public ./bin/www",
+    "startBackend": "node ./bin/www",
```

Comment:
the new commands might be a bit confusing, but it's basically the same as before.  Instead of running the frontend on its own port, the app now always builds out the `bundle` files.  When you run the `start` command, the app will start the backend server, which hosts all the static files of the DTR app, and the `buildFrontend` command will run and build everything out like it would in a production build.

The only difference now is that in a development environment, webpack will "watch" the files and rebuild them when there's a change, and the backend server will reload to serve up the new files with `nodemon`.  In a production environment, we just build out the frontend files like normal and serve them.

Reply:
When I run "npm start" in a Windows Command Prompt terminal, I would get this error: ERROR: Task not found: "VERSION=Dev".